bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -1134,14 +1134,19 @@ lnd/clm2/surfdata_map/landuse.timeseries_ne30np4_hist_16pfts_Irrig_CMIP6_simyr18
 <light_res    use_cn=".true." fire_method="nofire" >none</light_res>
 <light_res    use_cn=".true." phys="clm4_5"        >94x192</light_res>
 <light_res    use_cn=".true." phys="clm5_0"        >94x192</light_res>
+<light_res    use_fates=".true." phys="clm5_0"     >360x720</light_res>
 
 <stream_year_first_lightng use_cn=".true."   >0001</stream_year_first_lightng>
 <stream_year_last_lightng  use_cn=".true."   >0001</stream_year_last_lightng>
+<stream_year_first_lightng use_fates=".true.">0001</stream_year_first_lightng>
+<stream_year_last_lightng  use_fates=".true.">0001</stream_year_last_lightng>
 
 <stream_fldfilename_lightng hgrid="94x192"    use_cn=".true."   >atm/datm7/NASA_LIS/clmforc.Li_2012_climo1995-2011.T62.lnfm_Total_c140423.nc</stream_fldfilename_lightng>
 <stream_fldfilename_lightng hgrid="360x720"   use_cn=".true."   >atm/datm7/NASA_LIS/clmforc.Li_2016_climo1995-2013.360x720.lnfm_Total_c160825.nc</stream_fldfilename_lightng>
+<stream_fldfilename_lightng use_fates=".true."   >atm/datm7/NASA_LIS/clmforc.Li_2016_climo1995-2013.360x720.lnfm_Total_c160825.nc</stream_fldfilename_lightng>
 
 <lightngmapalgo use_cn=".true."                               >bilinear</lightngmapalgo>
+<lightngmapalgo use_fates=".true."                            >nn</lightngmapalgo>
```

Comment:
I used nearest neighbor for the lightning regridding because @lmkueppers' group runs mostly local and regional simulations at resolutions that are (much) higher than the input data. I will let @lmkueppers and others offer feedback on this.

Reply:
I don't think this matters so much for point scale simulations. 
At regional scale, there may be alternate resolution datasets to the current lightening one, so regridding seems preferable. 
@jkshuman is running at continental scales with fire, and we presumably will run FATES globally with fire on at some point.

Reply:
I will defer to @ekluzek on this point.

Reply:
OK change it to bilinear for FATES. Scientists can change it to nn on their own if it's appropriate for what they are doing. Likely that would just be if they have actual lightning data from some nearby source.

Reply:
Ok, done.